cfme/tests/v2v/test_csv_import.py

```diff
@@ -57,8 +62,8 @@ def migration_plan(appliance, infra_map, csv=True):
     return view
 
 
-def import_and_check(appliance, infra_map, error_text=None,
-                     filetype='csv', content=False, table_hover=False, alert=False):
+def import_and_check(appliance, infra_map, error_text=None, filetype='csv', content=False,
```

Comment:
Just a general comment, not specific to the PR.

This seems like something that should be a method on the MigrationPlan object, maybe with the 'check' side left to a test function.

Reply:
:D I might picked up wrong function name here, we are checking condition in test function only https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/v2v/test_csv_import.py#L244

Reply:
Updated "check" section in test :+1: